key-pkg-ver-3/create_kpkg.py

```diff
@@ -0,0 +1,83 @@
+from utils.create_kpkg_helper import *
+from utils.input_utils import *
+import pyinputplus as pyip, sys, json, re
+
+def main():
+    """Interactive function for creating key packages
+    """
+
+    target_list = None
+    operation = None
+    key_package = None
+    key_packages_list = []
+    first_user_input = "CREATE A NEW KEY PACKAGE"
+
+    while True:
+        if first_user_input == "CREATE A NEW KEY PACKAGE" :
+            # create a new key pkg
+            target_list = pyip.inputMenu(VALID_TARGET_LIST, numbered = True, prompt = "Please select a target list:\n")
+            operation = pyip.inputMenu(VALID_OPERATIONS, numbered = True, prompt = "Please select an operation:\n")
+            key_type = pyip.inputMenu(VALID_KEY_TYPES, numbered = True, prompt = "Please select a key type:\n")
+            key_name = input_key_name()
```

Comment:
Key_name, key_metadata ..etc should be input validated. And their size checked to be within limits as per the old version of the script.